packages/server/src/utils.ts

```diff
@@ -0,0 +1,74 @@
+import sha256 from "crypto-js/sha256";
+import {AuthType, KeyType, Property, PropertyType, Signature} from "@0auth/message";
+import {ec as ECDSA, eddsa as EdDSA} from "elliptic";
+
+export function utf8ToBase64(str: string): string {
+  return Buffer.from(encodeURIComponent(str), "binary").toString("base64");
+}
```

Comment:
Because `btoa` function cannot be used in node.js, `Buffer.from(...).toString("base64")` is used.
encodeURIComponent is used for Korean language processing.